src/app/activity/bounty/Paid.ts

```diff
@@ -0,0 +1,180 @@
+import { PaidRequest } from '../../requests/PaidRequest';
+import DiscordUtils from '../../utils/DiscordUtils';
+import Log, { LogUtils } from '../../utils/Log';
+import { GuildMember, MessageEmbed, Message, TextChannel } from 'discord.js';
+import MongoDbUtils from '../../utils/MongoDbUtils';
+import mongo, { Db, UpdateWriteOpResult } from 'mongodb';
+import { BountyCollection } from '../../types/bounty/BountyCollection';
+import { CustomerCollection } from '../../types/bounty/CustomerCollection';
+import RuntimeError from '../../errors/RuntimeError';
+import { BountyStatus } from '../../constants/bountyStatus';
+import { BountyEmbedFields } from '../../constants/embeds';
+
+
+export const paidBounty = async (request: PaidRequest): Promise<void> => {
+
```

Comment:
Let's have a log statement here with something like 'entering paid activity`

Reply:
There is one there, just a few lines down. Do you want it moved up?

Reply:
No, since it relies on a call that may error out. I'd like all of our activities to have a debug log at the very top that doesn't require any API (either explicitly or implicitly through discord js) or db calls. 

If you think that's too much, that's fair. Just thinking back to the last time I was going through the logs and I made a note to implement this in the future

Reply:
Done